bindings/python/pydeck/docs/deck.md

```diff
@@ -0,0 +1,101 @@
+pydeck.Deck API Documentation
+==========
+
+## pydeck.Deck
+
+```python
+class pydeck.*Deck*(
+    layers=[],
+    views=[pydeck.View()],
+    map_style="mapbox://styles/mapbox/dark-v9",
+    mapbox_key=None, 
+    initial_view_state=pydeck.ViewState()
+)
+```
+
+Constructor for a Deck object, similar to the [Deck](https://deck.gl/#/documentation/deckgl-api-reference/deck) class from deck.gl
+
+Requires a Mapbox API token to display a basemap, see notes below.
+
+#### Parameters
+
+`layers` : `pydeck.Layer` or list of pydeck.Layer`, default `[]`
+    pydeck.Layer objects to render
+
+`views` : `list` of `pydeck.View`, default `[pydeck.View()]`
+    List of `pydeck.View` objects to render. If rendering a standard map, there is rarely a reason to modify this.
+
+`map_style` : `str`, default `"mapbox://styles/mapbox/dark-v9"`
+    URI for Mapbox basemap style
+
+`mapbox_key` : `str`, default None
```

Comment:
- Ideally the param names can align with the standalone bundle API (`mapboxApiAccessToken`)
- How do you disable map?